apps/openmw/mwrender/actoranimation.cpp

```diff
@@ -51,6 +64,260 @@ ActorAnimation::~ActorAnimation()
     {
         mInsert->removeChild(iter->second);
     }
+
+    mScabbard.reset();
+}
+
+PartHolderPtr ActorAnimation::getWeaponPart(const std::string& model, const std::string& bonename, bool enchantedGlow, osg::Vec4f* glowColor)
+{
+    osg::Group* parent = getBoneByName(bonename);
+    if (!parent)
+        return NULL;
```

Comment:
Use nullptr please.

Reply:
Unfortunately, we already use NULL in animation code everywhere.
Should we create a separate pull request to replace all NULLs in OpenMW to nullptr?